core/src/components/refresher-content/refresher-content.tsx

```diff
@@ -127,7 +127,7 @@ export class RefresherContent implements ComponentInterface {
         <div class="refresher-refreshing">
           {this.refreshingSpinner && (
             <div class="refresher-refreshing-icon">
-              <ion-spinner name={this.refreshingSpinner}></ion-spinner>
+              <ion-spinner name={this.refreshingSpinner} aria-label="refreshing icon"></ion-spinner>
```

Comment:
`ion-refresher-content` is [failing](https://dequeuniversity.com/rules/axe/4.8/aria-progressbar-name?application=playwright) to pass Axe testing due to not passing an `aria-label` to `ion-spinner`.